hal/windows/UartWindows.hpp

```diff
@@ -49,7 +49,7 @@ namespace hal
                 space = SPACEPARITY
             };
 
-            UartWindowsConfig() = default;
+            UartWindowsConfig(){}
```

Comment:
Can you add a space please.

```suggestion
            UartWindowsConfig() {}
```